File: Design and Programming Environments/Iteration 3/TicketManager/Networking/src/main/java/ObjectProtocol/UpdateEventRequest.java
package ObjectProtocol;


import DTOPackage.EventDTO;

public class UpdateEventRequest implements Request{
    private EventDTO eventDTO;

    public EventDTO getEventDTO() {
        return eventDTO;
    }

    public UpdateEventRequest(EventDTO eventDTO) {
        this.eventDTO = eventDTO;
    }
}

File: Second year/Design and Programming Environments/TicketManagerCSharp/Networking/Requests/UpdateEventRequest.cs
using Networking.DTOS;
using System;

namespace Networking.Requests
{
    [Serializable]
    internal class UpdateEventRequest : Request
    {
        private EventDTO eventDTO;

        public UpdateEventRequest(EventDTO eventDTO)
        {
            this.eventDTO = eventDTO;
        }

        public EventDTO EventDTO { get => eventDTO; set => eventDTO = value; }
    }
}